Homework_8/Task_3.py

```diff
@@ -0,0 +1,9 @@
+result_link = 'https://www.youtube.com/watch?v=some_id&list=some_list&index=6&t=0s'
+dict_count = {}
+
+for dict in result_link:
```

Comment:
You shouldn't name variable dict. This name is reserved because this name for data type